Objects/interface/EventBase.h

```diff
@@ -71,6 +72,12 @@ namespace panda {
     //! Trigger object collection with additional features.
     HLTObjectStore triggerObjects = HLTObjectStore("triggerObjects");
 
+    //! Repeatable random number generator
+    RRNG* rng{0};
```

Comment:
My original reasoning was to avoid constructing `RRNG` if people don't want to use it. It sounds like only a fraction of the users care about it, and I'd like to avoid adding overhead for everyone else.

Reply:
The overhead is tiny, and if some new user tries to access a method through a null pointer, they'll just get a very unhelpful seg fault message.

Reply:
I'll let @yiiyama decide if this should always be instantiated or not. I think he's the only maintainer who won't use this feature.

Reply:
I think either is fine - on one hand we do instantiate things that not everyone always use (HLTObjects etc.), but on the other hand if the user has to call `initRNG()` anyway to start using this interface, we might as well instantiate it there.